src/aci_docs_sample.py

```diff
@@ -129,9 +129,9 @@ def create_container_group(aci_client, resource_group,
                            ip_address=group_ip_address)
 
     # Create the container group
-    aci_client.container_groups.create_or_update(resource_group.name,
+    aci_client.container_groups.begin_create_or_update(resource_group.name,
                                                  container_group_name,
-                                                 group)
+                                                 group).result()
 
     # Get the created container group
     container_group = aci_client.container_groups.get(resource_group.name,
```

Comment:
we need to update `requirements.txt` about version of azure-mgmt-containerinstance